src/main/java/com/johan/blignaut/entelect/vitalityincubator/task2/exception/CustomInternalServerException.java

```diff
@@ -0,0 +1,19 @@
+package com.johan.blignaut.entelect.vitalityincubator.task2.exception;
+
+public class CustomInternalServerException extends RuntimeException {
```

Comment:
Have a look at the `@ResponseStatus` annotation. If you use it on this class, you can throw a `CustomInternalServerException` and then spring will handle the error response for you, removing the need for the `CustomApiExceptionHandler`

Reply:
Alright, but then the custom message gets lost, doesn't it? But will do.

Reply:
No, you can still include the custom message by passing it as a parameter to the constructor